src/System.Text.Json/tests/Serialization/PolymorphicTests.cs

```diff
@@ -69,51 +69,47 @@ public static void ArrayAsRootObject()
             var dictionary = new Dictionary<string, string> { { "City", "MyCity" } };
             var arrayWithDictionary = new object[] { 1, true, dictionary, null, "foo" };
             json = JsonSerializer.ToString(arrayWithDictionary);
-            Assert.Equal(ExpectedJson, json);
-
-            json = JsonSerializer.ToString<object>(array);
```

Comment:
Are the "object" \ polymorphic tests handled elsewhere? It's an important feature for serialization today (not yet supported for deserialization)

Reply:
Oversight - I'll add them back.